mailpoet/views/deactivationPoll/index.html

```diff
@@ -0,0 +1,51 @@
+<div class="mailpoet-deactivate-survey-modal" id="mailpoet-deactivate-survey">
+  <div class="mailpoet-deactivate-survey-wrap">
+    <div class="mailpoet-deactivate-survey">
+
+      <script type="text/javascript">
+        window.addEventListener('load', function(event) {
+          var deactivateLink = document.querySelector('#the-list [data-slug="mailpoet"] span.deactivate a');
+          var overlay = document.querySelector('#mailpoet-deactivate-survey');
```

Comment:
Can we add an early return here if `deactivateLink` does not exist?

Reply:
Yes, we can. Good point.